code/game/objects/items/weapons/material/twohanded.dm

```diff
@@ -106,7 +107,8 @@
 	base_icon = "spearglass"
 	name = "spear"
 	desc = "A haphazardly-constructed yet still deadly weapon of ancient design."
-	force = WEAPON_FORCE_PAINFUL
+	force = WEAPON_FORCE_ROBUST
+	armor_penetration = 20 // It's a SPEAR!
```

Comment:
All of the armor penetration values needs their own defines as well.

Reply:
*painscream

Yes master... ;_;

Reply:
This armor penetration value should respect material hardness, I don't see how a piece of window can penetrate a armor vest better than a bullet.

Reply:
Let's don't make it more complicated then it should be, @Kurgis.